_resources/participatory-design-inclusivity-heuristics.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: heuristics
+title: Participatory design&#58; inclusivity heuristics
+permalink: /resources/participatory-design-inclusivity-heuristics/
+redirect_from:
+  - /participatory-design-inclusivity-heuristics/
+description: A set of heuristics for ensuring that a participatory design effort is proceeding in an inclusive way.
+---
+
+<h1>Participatory Design: Inclusivity Heuristics</h1>
+
+<p>The following indicators can be used to help determine if a <a href="{{site.baseurl}}/fundamentals/participatory-design">participatory design</a> process is proceeding in a way that encourages input from stakeholders and users. Indicators are spread across three broad categories: <a href="#account-for-context">Account for context</a>, <a href="#encourage-participation">Encourage participation</a>, and <a href="#inform-and-share-understanding">Inform and share understanding</a>.</p>
+
+<table>
+  <thead>
+    <tr>
+      <th scope="col" class="col-indicator">Indicator</th>
+      <th scope="col" class="col-why-it-matters">Why it matters</th>
+      <th scope="col" class="col-good-signs">Good signs</th>
+      <th scope="col" class="col-warning-signs">Warning signs</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th colspan="4" id="account-for-context"><p>Account for context</p></th>
+    </tr>
+    <tr>
+      <th scope="row" class="col-indicator"><p>Values</p></th>
+      <td class="col-why-it-matters"><p>Values tie people together in ways that are difficult to break, especially when they’re not made explicit. Accounting for values helps teams approach design honestly and respectfully.</p></td>
+      <td class="col-good-signs"><p>The team identifies and reflects on its values. The team respects the values of the people affected by its work. The team acknowledges when it seeks to challenge the values of others.</p></td>
+      <td class="col-warning-signs"><p>The team doesn’t identify or discuss its values. The team does not see or consider the merits of the values of others. The team misleads others when attempting to change their values.</p></td>
```

Comment:
How would a team who is doing any of these things come to realize that they are doing them? What kind of activities are typical when a team is misleading others?

Reply:
Awareness and reflection are key, which is partly why I'm so adamant that this document exists! A team is very likely misleading others if it talks about something but doesn't actually do it ("vaporware"). I'm specifically thinking of [the difference between verbalism and activism](https://blogs.lt.vt.edu/sarag1/2017/10/14/activism-verbalism-praxis/) that Paulo Freire describes in Pedagogy of the Oppressed. Technologists need to consider both of these when thinking about how our work liberates/oppresses.